app/models/orphan.rb

```diff
@@ -175,16 +183,14 @@ def orphan_born_long_after_fathers_death?
   end
 
   def can_be_inactivated
-    if currently_sponsored?
-      errors.add :orphan_status,
+    if sponsored? && being_inactivated?
```

Comment:
Readability is important.  Usually, if you need to add comments in ruby, something else may be amiss